### DISCUSSION TITLE
guestbook

### DISCUSSION THREAD

**Discussion by giscus** [nathanbuchar/n8.engineer#11] [Comments]
# guestbook

https://n8.engineer/guestbook

**Comment by nathanbuchar** [comment]
Welcome! 👋

**Comment by AsaAyers** [comment]
Is this also visible somewhere on Github since it's powered by giscus?

**Reply by nathanbuchar** [reply]
Right here! I don't surface post replies on the site, just top-level comments.

**Comment by nathanbuchar** [comment]
Yep! Discussions on [nathanbuchar/n8.engineer](https://github.com/nathanbuchar/n8.engineer/discussions/categories/comments)

**Comment by tif-calin** [comment]
hello from sacramento

**Comment by leoncesaba-ctrl** [comment]
thank you

**Comment by jpetkauskas** [comment]
Hey Nate! You've got an awesome site here. All the best!
